packages/ui/src/theme/css/component/button.scss

```diff
@@ -225,11 +232,25 @@
       --amplify-components-button-large-padding-inline-end
     );
   }
-}
+  &--disabled {
+    background-color: var(--amplify-internal-button-disabled-background-color);
+    border-color: var(--amplify-internal-button-disabled-border-color);
+    color: var(--amplify-internal-button-disabled-color);
+    text-decoration: var(--amplify-internal-button-disabled-text-decoration);
+    cursor: not-allowed;
+  }
 
-.amplify-button__loader-wrapper {
```

Comment:
Why was underscore used instead of hyphen?

Reply:
BEM syntax follows the guide of .{Block}__{Element}--{Modifier} the underscores are saying the `loader-wrapper` is an element of the `amplify-button` block.  Its just the naming convention.